mobify/platform-scaffold
# WEB-1357: Implement header search

**JIRA**: https://mobify.atlassian.net/browse/WEB-1357
**Linked PR**: https://github.com/mobify/progressive-web-sdk/pull/605

## Changes
- Implement header search

## Todo
- ~~Ensure the user can close the modal by including the native close icon in the strip above the keyboard (iOS). Not sure if there is an Android equivalent but we should use it if so~~
- [x] Maintain a search icon in the top right of the UI
- [x] When text is input there should be a X icon to clear the text

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Type 'eye' in search input, you should see autocomplete dropdown
  - Click on autocomplete item to search for that term
  - or press enter/go on your keyboard to search for current input query text
